cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2020-04-01T00:00:00Z
+index-state: 2019-06-20T00:00:00Z
```

Comment:
What? Why?

Reply:
Not sure why, but Sam already fixed it in a later commit.